driver-core/src/test/functional/com/mongodb/client/model/expressions/DateExpressionsFunctionalTest.java

```diff
@@ -31,12 +31,13 @@ class DateExpressionsFunctionalTest extends AbstractExpressionsFunctionalTest {
     // https://www.mongodb.com/docs/manual/reference/operator/aggregation/#date-expression-operators
 
     private final ZonedDateTime utcDateTime = ZonedDateTime.ofInstant(Instant.parse("2007-12-03T10:15:30.005Z"), ZoneId.of(ZoneOffset.UTC.getId()));
+    private final StringExpression utc = of("UTC");
+    private final DateExpression utcDateEx = of(utcDateTime.toInstant());
```

Comment:
We shouldn't call it `utcDateEx` because (unlike Java's `utcDateTime`) it's time zone–agnostic (it's semantically an instant). Let's call it `dateEx`?